arch/arc/Kconfig

```diff
@@ -9,6 +9,12 @@ menu "ARC Options"
 config ARCH
 	default "arc"
 
+config ZEPHYR_TOOLCHAIN_VARIANT
+	string "ZEPHYR_TOOLCHAIN_VARIANT"
+	default "$(ZEPHYR_TOOLCHAIN_VARIANT)"
+	help
+	  zephyr_toolchain_variant get from env
+
```

Comment:
Any explanation about why do we need this?

ZEPHYR_TOOLCHAIN_VARIANT is used directly in other places:
```
config TOOLCHAIN_SUPPORTS_THREAD_LOCAL_STORAGE
	bool
	default y if "$(ZEPHYR_TOOLCHAIN_VARIANT)" = "zephyr"
	help
	  Hidden option to signal that toolchain supports generating code
	  with thread local storage.
```

Reply:
Thanks for your point, I should use `"$(ZEPHYR_TOOLCHAIN_VARIANT)"`, not` $(ZEPHYR_TOOLCHAIN_VARIANT)`